_config/email.yml

```diff
@@ -1,12 +1,55 @@
 ---
-Name: emailconfig
+Name: mailer
 ---
 SilverStripe\Core\Injector\Injector:
-  Swift_Transport: Swift_MailTransport
-  Swift_Mailer:
-    constructor:
-      - '%$Swift_Transport'
-  SilverStripe\Control\Email\Mailer:
-    class: SilverStripe\Control\Email\SwiftMailer
-    properties:
-      SwiftMailer: '%$Swift_Mailer'
+  Symfony\Component\Mailer\MailerInterface:
+    class: Symfony\Component\Mailer\Mailer
+    constructor:
+      0: '%$Symfony\Component\Mailer\Transport\TransportInterface'
+      1: '%$Symfony\Component\Messenger\MessageBusInterface.mailer'
+  Symfony\Component\Messenger\MessageBusInterface.mailer:
+    class: Symfony\Component\Messenger\MessageBus
+    constructor:
+      0:
+        - '%$Symfony\Component\Messenger\Middleware\MiddlewareInterface.mailer'
+  Symfony\Component\Messenger\Middleware\MiddlewareInterface.mailer:
+    class: Symfony\Component\Messenger\Middleware\SendMessageMiddleware
+    constructor:
+      0: '%$Symfony\Component\Messenger\Transport\Sender\SendersLocatorInterface.mailer'
+      1: '%$Symfony\Component\EventDispatcher\EventDispatcherInterface.mailer'
+  Symfony\Component\Messenger\Transport\Sender\SendersLocatorInterface.mailer:
+    class: Symfony\Component\Messenger\Transport\Sender\SendersLocator
+    constructor:
+      0:
+        '*':
+          - 'Symfony\Component\Messenger\Transport\Sender\SenderInterface.mailer'
+      1: '%$Symfony\Component\DependencyInjection\ContainerInterface.mailer'
+  Symfony\Component\Messenger\Transport\Sender\SenderInterface.mailer:
+    class: SilverStripe\Control\Email\EmailSender
+  Symfony\Component\DependencyInjection\ContainerInterface.mailer:
+    class: Symfony\Component\DependencyInjection\Container
+    calls:
+      - ['set', ['Symfony\Component\Messenger\Transport\Sender\SenderInterface.mailer', '%$Symfony\Component\Messenger\Transport\Sender\SenderInterface.mailer']]
+  Symfony\Component\EventDispatcher\EventDispatcherInterface.mailer:
+    class: Symfony\Component\EventDispatcher\EventDispatcher
+---
+Name: mailer-dsn-default-config
+---
+SilverStripe\Core\Injector\Injector:
+  Symfony\Component\Mailer\Transport\TransportInterface:
+    factory: Symfony\Component\Mailer\Transport
+    factory_method: fromDsn
+    constructor:
+      dsn: 'sendmail://default'
```

Comment:
Nope, what's in installer is to copy in some yaml to use `SendMailTransport` transport for _new_ CMS 4 projects

Existing CMS 4 projects stay on the default, deprecated `MailTransport`.  When they run `composer update` they stay on `MailTransport` - this was done so that nothing would break on upgrade

In CMS 5 there is no support at all for MailTransport, so we default to the `sendmail` DSN in framework